.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+# Xi-Editor Issue Template
+- [ ] I have searched existing issues and could not find my issue.
+- [ ] I have studied the documentation.
+- [ ] This is not a personal support question, which belongs on #xi (irc.mozilla.org)
```

Comment:
I think 'personal support' is a bit vague. If you can't get xi to compile, I think opening an issue is a reasonable step. Encouraging people to go to IRC is good, but I'm not sure it should be mandatory?

Reply:
That's 👍 , I had just looked at other examples that said something like this, but I have no strong feelings on the subject.